### PR TITLE
updated date-picker to have initial date if there

### PR DESCRIPTION
**Pull request by kialj876**
Signed-off-by: Kial Jinnah <kialj876@gmail.com>

*Issue #:* /bcgov/entity#10059

*Description of changes:*
- initial value is populated now
- clearable event on the text field will also clear the date ~(waiting on shared comp merge)~ done

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/components/common/CourtOrder.vue [hunk 1]
```diff
@@ -138,6 +138,8 @@
               title="Select the date of the order"
               clearable
               :errorMsg="errors.orderDate.message ? errors.orderDate.message : ''"
+              :initialValue="orderDate"
```
Comment (jordiwes) [inline ppr-ui/src/components/common/CourtOrder.vue:141]: Tested in the date picker component?

Comment (kialj876) [reply ppr-ui/src/components/common/CourtOrder.vue:141]: yep -- same with the other changes

Comment (kialj876) [reply ppr-ui/src/components/common/CourtOrder.vue:141]: just had that merged/published in the newer version

**Review by doug-lovett — APPROVED** [review]
Looks good.

File: ppr-ui/src/components/common/CourtOrder.vue (hunk 1 above)
Comment (jordiwes) [reply ppr-ui/src/components/common/CourtOrder.vue:141]: cool

**Review by jordiwes — APPROVED** [review]

**Pull request merged**